packages/client/e2e/credit.spec.ts

```diff
@@ -81,15 +82,41 @@ test.describe("Credit", () => {
         .first()
         .locator("div")
         .first()
-      // Wait for first quote response
-      await expect(firstQuote).not.toContainText("Awaiting response", {
-        timeout: Timeout.LONG,
-      })
-
-      await firstQuote.hover()
 
-      await firstQuote.getByText(/Accept/).waitFor({state: "visible"})
-      await firstQuote.getByText(/Accept/).click()
+      // retry logic to circumvent intermittent failures at clicking on the accept button
+      const MAX_ATTEMPT = 2
+      let isAcceptBtnClicked = false
+
+      const acceptQuote = async () => {
```

Comment:
added logic to catch any error while clicking on accept button and capture state in a variable to check for retry